<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>memoryWindow</class>
 <widget class="QDialog" name="memoryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>696</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>697</width>
    <height>1024</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Memory</string>
  </property>
  <widget class="QCheckBox" name="realtimeLoadCheckBox">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Realtime load</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="readMemoryPushButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>450</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Read memory</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshWindowPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Refresh window</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshCompareWindowPushButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>450</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Refresh &amp; compare window</string>
   </property>
  </widget>
  <widget class="QPushButton" name="compareWindowPushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>450</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Compare window</string>
   </property>
  </widget>
  <widget class="QComboBox" name="memoryTypeComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="writeMemoryPushButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>450</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Write memory</string>
   </property>
  </widget>
  <zorder>memoryTypeComboBox</zorder>
  <zorder>realtimeLoadCheckBox</zorder>
  <zorder>refreshWindowPushButton</zorder>
  <zorder>compareWindowPushButton</zorder>
  <zorder>refreshCompareWindowPushButton</zorder>
  <zorder>readMemoryPushButton</zorder>
  <zorder>writeMemoryPushButton</zorder>
 </widget>
 <tabstops>
  <tabstop>memoryTypeComboBox</tabstop>
  <tabstop>realtimeLoadCheckBox</tabstop>
  <tabstop>refreshWindowPushButton</tabstop>
  <tabstop>compareWindowPushButton</tabstop>
  <tabstop>refreshCompareWindowPushButton</tabstop>
  <tabstop>readMemoryPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
